web-console/5-components/wizard-results.md

```diff
@@ -0,0 +1,66 @@
+# Wizard Results
+
+The results step for the "add to projects" wizards should follow a consistent format while supporting the unique needs of the various methods by which users may add resources to their projects. The experience should be similar, even if the types of information to be displayed is different.
+
+## Design
+
+![results steps callouts](img/results-step-callouts.png)
+
+1. A status icon followed by the status. The status should call out the resource being created by name and if the action is initiated from outside of a project, the project name should be included as well.
+  - Status icons
+    - `fa-clock-o` for provisioning
+    - `pficon-ok` for success
+    - `pficon-error-circle-o` for failure
+1. Additional details about the status, such as error messages, additional actions to be performed or estimates for how long the operation may take.
+1. An action to take users to the overview of the project that the operation affects.
+1. Other information specific to the type of operation, for example, template messages, applied parameters, documentation, etc.
+
+
+## Implementation details
+
+### Provisioned Services Without Bindings
+![results from a provisioned service with no binding](img/results-provisioned-service-no-binding.png)
+1. When a provision is in progress, the status should read "X is being provisioned in Y Project" where X is the Service Class Display Name. Other statuses should read similarly: "...has been provisioned successfully..." or "...failed to provision..."
+1. The UI should give the user an indication of how long the provision may take. If no specific message is provided by the broker, the default should set the user's expectation, but need not be overly specific, for example "This may take several minutes."
+  - There is an [issue](https://github.com/openservicebrokerapi/servicebroker/issues/315) open to the Open Service Broker to allow brokers to return a message for async operations before polling. 
+1. Links to documentation, support and a dashboard should be provided if those links exist (services may not provide all three items).
+
+![results provisioned service fail](img/results-provisioned-service-fail.png)
+
+If a service failed to provision, users should be given:
+1. The relevant error message.
```

Comment:
Curious if this error message should be contained somehow, similar to the box you have around the template message when a template is instantiated. Just thinking this could get pretty long so somehow sectioning off the details might be worth considering.

Reply:
I hadn't thought of that. FWIW, there is work upstream to improve the formatting of the error messages that come back from the server. But putting a box around could help separate things out.

Reply:
I think that's a good idea. That way it's extra clear which parts came from this specific thing vs what is always there

Reply:
This is very much related to this issue: https://github.com/openshift/origin-web-console/issues/1924#issuecomment-328188221
 
I think there's a larger story around how we deal with those gnarly server errors.